share/completions/ansible.fish

```diff
@@ -0,0 +1,106 @@
+# fish 2.2.0 does not include native snippet support. Upgrade to fish >= 2.3.0
+# or append the following code to your ~/.config/fish/config.fish.
+
+# for file in ~/.config/fish/conf.d/*.fish
+#     source $file
+# end
+# ansible
+complete -c ansible -s a -l args -r -f -d "module arguments"
+complete -c ansible -l ask-vault-pass -f -d "ask for vault password"
+complete -c ansible -s B -l background -r -f -d "run asynchronously, failing after X seconds (default=N/A)"
```

Comment:
"failing after X seconds" - does that mean that this takes an argument? "default=N/A" - can that argument be the string "N/A"?